docs/prod-host-setup.md

```diff
@@ -240,201 +240,65 @@ process via the following tools:
 
-Verification can be done by running:
+Verify that swap is disabled by running:
 
 ```bash
-grep -q "^(Mitigation: CSV2, BHB|Not affected)$" \
-/sys/devices/system/cpu/vulnerabilities/spectre_v2 && \
-echo "SPECTRE V2 -> OK" || echo "SPECTRE V2 -> NOT OK"
+grep -q "/dev" /proc/swaps && \
+echo "swap partitions present (Recommendation: no swap)" \
+|| echo "no swap partitions (OK)"
 ```
 
-##### Bounds Check Bypass Store (Spectre V1)
+### Mitigating hardware vulnerabilities
 
-Verification for mitigation against Spectre V1 can be done:
+> **Note** Firecracker is not able to mitigate host's hardware vulnerabilities.
+Adequate mitigations need to be put in place when configuring the host.
 
-```bash
-grep -q "^(Mitigation:|Not affected)$" \
-/sys/devices/system/cpu/vulnerabilities/spectre_v1 && \
-echo "SPECTRE V1 -> OK" || echo "SPECTRE V1 -> NOT OK"
-```
+> **Note** Firecracker is designed to provide isolation boundaries between
+microVMs running in different Firecracker processes. It is strongly recommended
+that each Firecracker process corresponds to a workload of a single tenant.
 
-##### [Intel only] Apply L1 Terminal Fault (L1TF) mitigation
+> **Note** For security and stability reasons it is highly recommended to load
+updated microcode as soon as possible. Aside from keeping the system firmware
+up-to-date, when the kernel is used to load updated microcode of the CPU this
+should be done as early as possible in the boot process.
 
-These features provide mitigation for Foreshadow/L1TF side-channel issue on
-affected hardware.
-They can be enabled by adding the following Linux kernel boot parameter:
+#### Side channel attacks
 
-```console
-l1tf=full,force
-```
+It is strongly recommended that users follow the
+[Linux kernel documentation on Spectre Side Channels](https://docs.kernel.org/admin-guide/hw-vuln/spectre.html)
+when configuring mitigations against "Spectre" attacks and
+[Linux kernel documentation on hardware vulnerabilities](https://docs.kernel.org/admin-guide/hw-vuln/index.html)
+for mitigations against other side channel attacks.
 
-which will also implicitly disable SMT.  This will apply the mitigation when
-execution context switches into microVMs.
-Verification can be done by running:
+Additionally users should consider:
 
-```bash
-declare -a CONDITIONS=("Mitigation: PTE Inversion" "VMX: cache flushes")
-for cond in "${CONDITIONS[@]}"; \
-do (grep -q "$cond" /sys/devices/system/cpu/vulnerabilities/l1tf && \
-echo "$cond: ENABLED (OK)") || \
-echo "$cond: DISABLED (Recommendation: ENABLED)"; done
-```
-
-See more details [here](https://www.kernel.org/doc/html/latest/admin-guide/hw-vuln/l1tf.html#guest-mitigation-mechanisms).
+- Enabling [Page Table Isolation](https://www.kernel.org/doc/html/next/x86/pti.html)
+  as a measure against some variants of "Meltdown" attacks
+- Disabling
+  [Kernel Samepage Merging](https://www.kernel.org/doc/html/latest/admin-guide/mm/ksm.html)
```

Comment:
Can we find a good source on why this should be disabled? I usually refer to the Flush+Reload paper, but I assume there's more recent guidance.

Reply:
added a link to the paper, not sure what the state-of-the-art guidance is